### PR TITLE
Add a lot of applications to Project Management

### PR DESCRIPTION
**Pull request by joubertredrat**
PR created used this form https://github.com/Kickball/awesome-selfhosted/issues/282

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -562,17 +562,21 @@ Some [Groupware](#groupware) solutions also feature file sharing synchronization
 See also [Ticketing](#ticketing), [Task management/To-do lists](#task-managementto-do-lists) and [Bug Trackers](#bug-trackers)*
 
   * [Apache Bloodhound](https://bloodhound.apache.org/) - Manage software products, Keep track of features, tasks and bugs `Apache` `Python`
+  * [Bonobo](https://bonobogitserver.com/) - An open-source, pure Java stack for managing, viewing, and serving Git repositories. ([Source code](https://github.com/jakubgarfield/Bonobo-Git-Server)) `MIT` `C#`
```
Comment (nodiscc) [inline README.md]: The description is wrong

File: README.md [hunk 2]
```diff
@@ -562,17 +562,21 @@ Some [Groupware](#groupware) solutions also feature file sharing synchronization
...
   * [GitLab](http://gitlab.org/) - Self Hosted Git Management Application - GitLab is open source software to collaborate on code. Manage git repositories with fine grained access controls that keep your code secure. Perform code reviews and enhance collaboration with merge requests. Each project can also have an issue tracker and a wiki ([Public instance](https://git.framasoft.org/))
   * [Gitlist](http://gitlist.org/) - A web-based git repository browser - GitList allows you to browse repositories using your favorite browser, viewing files under different revisions, commit history and diffs. ([Source code](https://github.com/klaussilveira/gitlist))
-  * [Gitprep](http://gitprep.org/) - Portable Github clone ([Source code](https://github.com/yuki-kimoto/gitprep), [Demo](http://perlcodesample.sakura.ne.jp/gitprep.cgi))
+  * [GitPrep](http://perlcodesample.sakura.ne.jp/gitprep-site/) - GitPrep is Github clone. you can install portable github system into unix/linux. You can create users and repositories without limitation. This is free software. ([Demo](http://perlcodesample.sakura.ne.jp/gitprep/gitprep.cgi), [Source code](https://github.com/yuki-kimoto/gitprep)) `Other` `Perl`
```
Comment (nodiscc) [inline README.md]: Previous description was fine

**Comment by nodiscc** [comment]
@joubertredrat please fix the 2 points mentioned above. Thanks for the patch

**Comment by joubertredrat** [comment]
I will make new PR

**Pull request closed without merging**